docs/docs/modules/models/llms/integrations.mdx

```diff
@@ -118,6 +118,73 @@ const res = await model.call(
 );
 ```
 
+You can also pass in the optional `returnPromptLayerID` string to get a promptLayerRequestID like below
+```typescript
+import { PromptLayerOpenAI } from "langchain/llms/openai";
+
+const model = new PromptLayerOpenAI({
+  temperature: 0.9,
+  openAIApiKey: "YOUR-API-KEY", // In Node.js defaults to process.env.OPENAI_API_KEY
+  promptLayerApiKey: "YOUR-API-KEY", // In Node.js defaults to process.env.PROMPTLAYER_API_KEY
+  returnPromptLayerID: true,
+});
+const res = await model.generate([
+  "What would be a good company name a company that makes colorful socks?"
+]);
+
+console.log(JSON.stringify(res, null, 3));
+// {
+//   "generations": [
+//      [
+//         {
+//            "text": " Socktastic!",
+//            "generationInfo": {
+//               "finishReason": "stop",
+//               "logprobs": null,
+//               "promptLayerRequestID": 2066417
+//            }
+//         }
+//      ]
+//   ],
+//   "llmOutput": {
+//      "tokenUsage": {
+//         "completionTokens": 5,
+//         "promptTokens": 23,
+//         "totalTokens": 28
+//      }
+//   }
+//}
+```
+
+### PromptLayerOpenAIChat
+PromptLayer is compatible with the OpenAIChat model as well. Here is an example of getting the PromptLayerOpenAIChat requestID:
```

Comment:
Remove, `OpenAIChat` is mostly deprecated in favor of the `ChatOpenAI` model (which you've added support to!).

Reply:
Add docs to `chat_models/integrations.mdx` instead. Also, we have been shifting to use imported examples as show here:

https://github.com/hwchase17/langchainjs/blob/main/.github/contributing/INTEGRATIONS.md#documentation-and-integration-tests